dashboard/src/main/webapp/src/pages/serverstatus/ActiveNodeListPage.vue

```diff
@@ -17,31 +17,46 @@
 
 <template>
   <div>
-    <el-table :data="pageData.tableData" height="550" style="width: 100%">
-      <el-table-column prop="id" label="Id" min-width="140" />
-      <el-table-column prop="ip" label="IP" min-width="80" />
+    <el-table
+      :data="pageData.tableData"
+      height="550"
+      style="width: 100%"
+      :default-sort="sortColumn"
+      @sort-change="sortChangeEvent"
+    >
+      <el-table-column prop="id" label="Id" min-width="140" :sortable="true" />
+      <el-table-column prop="ip" label="IP" min-width="80" :sortable="true" />
       <el-table-column prop="grpcPort" label="GrpcPort" min-width="80" />
       <el-table-column prop="nettyPort" label="NettyPort" min-width="80" />
-      <el-table-column prop="usedMemory" label="UsedMem" min-width="80" :formatter="memFormatter" />
+      <el-table-column
+        prop="usedMemory"
+        label="UsedMem"
+        min-width="80"
+        :formatter="memFormatter"
+        :sortable="true"
+      />
       <el-table-column
         prop="preAllocatedMemory"
-        label="PreAllocatedMem"
+        label="Pre-Mem"
```

Comment:
I think the original name is better.

Reply:
> I think the original name is better.

It turned out that the name was too long, I had to change lines when the page was displayed, and I looked very uncomfortable.Do you have any good ideas?

Reply:
Maybe we can fix the first column.
https://element.eleme.io/#/en-US/component/table#table-with-fixed-column

Reply:
done